src/parser/demonhunter/havoc/modules/talents/GlaiveTempest.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import TalentStatisticBox from 'interface/others/TalentStatisticBox';
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import STATISTIC_ORDER from 'interface/others/STATISTIC_ORDER';
+import { formatThousands } from 'common/format';
+import Events from 'parser/core/Events';
+
+class GlaiveTempest extends Analyzer {
+
+    damage = 0;
+
+    constructor(...args) {
+        super(...args);
+        this.active = this.selectedCombatant.hasTalent(SPELLS.GLAIVE_TEMPEST_TALENT.id);
+        if (!this.active) {
+          return;
+        }
+        this.addEventListener(Events.damage.by(SELECTED_PLAYER).spell(SPELLS.GLAIVE_TEMPEST_DAMAGE), this.onDamageEvent);
+      }
+
+    onDamageEvent(event) {
+        this.damage += event.amount;
```

Comment:
probably want to count damage that was absorbed as well

this.damage += event.amount + {event.absorb || 0);

Reply:
Fixed